ocdskingfisherarchive/archive.py

```diff
@@ -0,0 +1,8 @@
+
+class Archive:
```

Comment:
Not sure we need a class (could just be a regular function), but fine for now since there's still a lot of code to write.

Reply:
Given external services this needs to work with, in order to test it well we'll need a lot of mocking, so I'm seeing the code for this broken into many functions to allow for good mocking. That's just always seemed easier to comprehend with objects - you can have some class state passed round to. But can always change later.